src/lib/actions-executed.state.ts

```diff
@@ -26,7 +26,8 @@ export class ActionsExecutedState implements NgxsOnInit, OnDestroy {
                         return;
                     }
 
-                    let count = getState()[actionType] || 0;
+                    var state = getState();
+                    let count = state ? state[actionType] || 0 : 0;
```

Comment:
hi, can you change it to optional chaining `let count = getState()?.[actionType] || 0;`

Reply:
Thank you for the hint @joaqcid, i fixed it according to your suggestion